test/parallel/test-timers-same-timeout-wrong-list-deleted.js

```diff
@@ -34,3 +34,9 @@ const handle1 = setTimeout(common.mustCall(function() {
   // correct list and refrain from removing the new TIMEOUT list which
   // contains the reference to the newer timer.
 }), TIMEOUT);
+
+function getActiveTimers() {
+  const activeHandles = process.getActiveHandles();
+  return activeHandles.filter((handle) => handle instanceof Timer);
+}
```

Comment:
It’s hard to tell for me where this function is being used … also, we’re kind of shifting our timers implementation to not use any handles anymore, /cc @apapirovski

Reply:
Should I remove it ?